translations/wesnoth-Flight_Freedom/ja.po

```diff
@@ -5554,7 +5553,6 @@ msgstr ""
 "分かったよ。じゃあな、Malakar さん。俺にも、「麻の縄」にぶら下がって踊る時が"
 "きたようだな..."
 
-# $po: The expression is too graphic.
```

Comment:
I agree, to be honest.

Reply:
I thought I had deleted that comment, but it remained...

Reply:
oh no it's ok, the red means a deletion. I was just commenting on one of the intermediate commits

Reply:
> oh no it's ok, the red means a deletion. I was just commenting on one of the intermediate commits

I forgot to organise the comments I wrote in ja.po and PR'd them, and all the history is now here as well. I have to be careful... lol

Reply:
It's ok, I squashed upon merging, so it won't show up in the target branch's history!